src/kajal/ScannerExamples/Assignment15.java

```diff
@@ -0,0 +1,158 @@
+package kajal.ScannerExamples;
+
+public class Assignment15 {
+
+	double totalBalanceIcici, totalBalanceHdfc, totalBalanceBob, totalBalanceSbi, totalBalance;
+	int debitCountIcici, debitCountHdfc, debitCountBob, debitCountSbi, creditCountIcici, creditCountHdfc,
+			creditCountBob, creditCountSbi, totalDebitCount, totalCreditCount;
+
+	void setBalance(double balace, String bankName) {
+
+		switch (bankName.toUpperCase()) {
+		case "ICICI":
+			totalBalanceIcici = balace;
+			break;
+			
+		case "HDFC":
+			totalBalanceHdfc = balace;
+			break;
+			
+		case "BOB":
+			totalBalanceBob = balace;
+			break;
+			
+		case "SBI":
+			totalBalanceSbi = balace;
+			break;
+			
+		default:
+			System.out.println("Invalid bank, Enter valid bank name");
+		}
+	}
+
+	void Operation(String op, String bankName, int amount) {
+		switch (op.toUpperCase()) {
+		default:
+			System.out.println("Invalid Transcation");
+		case "DEBIT":
+			switch (bankName.toUpperCase()) {
+
+			default:
+				System.out.println("Invalid bank");
+				break;
+				
+			case "ICICI":
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.

Code snippet

int debit(int totalAmt, int debitAmt) {
		if (totalAmt < debitAmt) {
			return 0;
		} else
			return totalAmt - debitAmt;
}

totalAmt will your iciciAmt (instance variable)